deepseq.cabal

```diff
@@ -53,6 +53,11 @@ library
     if impl(ghc < 7.6)
       build-depends: ghc-prim == 0.2.*
 
+  if impl(ghc >= 7.8)
+    other-extensions:
+      ScopedTypeVariables
+      Trustworthy
```

Comment:
It's not really important. `other-extensions` is only informative; it doesn't actually enable anything. That's just saying that some modules use those extensions with those versions. I can step them out for sure.